settings/table-string.php

```diff
@@ -141,7 +141,9 @@ public function column_translations( $item ) {
 				esc_attr( $key ),
 				esc_attr( $item['row'] ),
 				esc_html( $languages[ $key ] ),
-				format_to_edit( $translation ) // Don't interpret special chars.
+				format_to_edit(
+					$translation === $item['string'] ? '' : $translation
```

Comment:
That means that if a user manually copy the original string in the translation, we will remove it.
Do we really want do that?

Reply:
Ah! You got me!
In fact I've done that to tests the behavior on the Pro.
But you're right to point it out, I don't think we should prevent anyone to duplicate a string on purpose. Don't you agree?

Reply:
This is what I had in mind.

Reply:
Done in 1cca4c43.
Actually we to "translate" entry when preparing items for the table, but not using `Translations::translate()` otherwise we always get the source if set to empty string in DB...

So to summarize I removed the code forcing to empty string on display and on save.
Then when preparing items for the strings table, ensure to display what's from DB.